### ISSUE TITLE
a11y chat help is missing important info

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#268345]
Such as how to navigate to the chat list

**Comment by meganrogge** [comment]
Yes, that exists in here `To focus the chat request and response list, invoke the Focus Chat command (Command+DownArrow). This will move focus to the most recent response, which you can then navigate using the up and down arrow keys.
To return to the last chat response you focused, invoke the Focus Last Focused Chat Response command (Shift+Command+DownArrow).`

However, seems it's reversed as that should only be showing `DownArrow` for quick chat, it should show `UpArrow` for the agent view.

<img width="754" height="429" alt="Image" src="https://github.com/user-attachments/assets/c103848f-0a28-4782-9772-3b208e78142c" />
